.travis.yml

```diff
@@ -26,4 +40,15 @@ after_script:
     - printf "$TRAVIS_COMMIT_LOG\n"
 
 after_success:
-    - if [[ ! -z "${CODECOV_TOKEN}" ]]; then (bash <(curl -s https://codecov.io/bash) || echo "Codecov did not collect coverage reports"); fi;
```

Comment:
Why did you remove this check? if CODECOV_TOKEN is removed from travis setting, the script would fail.

Reply:
As I realized it needed only for private repos. As you can see it works w/o token now.

Reply:
Right on. I was confused too.